### PR TITLE
Jorge/delete itm

### PR DESCRIPTION
**Pull request by Luisjor**

### CODE REVIEW

**Review by svaleria — APPROVED** [review]

File: 3.Tests/UI/Features/Item/Update.feature [hunk 1]
```diff
@@ -4,9 +4,9 @@ Feature: Item Update
     Background:
         Given the user is logged in
 
-    @Smoke @Regression @create.project.NewProject @create.item
+    @Smoke @Regression @create.project.Exercise @create.item @update
```
Comment (sebasonti-jalauni) [inline 3.Tests/UI/Features/Item/Update.feature:7]: Is the last tag @update supposed to be the id of this test? if so it should be more descriptive, like @UI_Update_Item

File: 2.Views/WebAppPages/HomePage.cs [hunk 2]
```diff
@@ -157,5 +175,11 @@ public class HomePage
             )
         );
 
+    public IElement ItemDeletedAlert() =>
+        new Button("", new Locator(LocatorType.XPath, $"//span[@id='InfoMessageText']"));
+
+    public IElement NoItemsOnMain() =>
```
Comment (sebasonti-jalauni) [inline 2.Views/WebAppPages/HomePage.cs:181]: Why is this a method? it should be a property right? same for ItemDeletedAlert

Comment (Luisjor) [reply 2.Views/WebAppPages/HomePage.cs]: Done, thank you

File: 3.Tests/UI/Features/Item/Update.feature (hunk 1 above)
Comment (Luisjor) [reply 3.Tests/UI/Features/Item/Update.feature]: Done, thank you

**Review by Schlvf — APPROVED** [review]

**Pull request merged**